src/mapeo-project.js

```diff
@@ -108,6 +111,7 @@ export class MapeoProject extends TypedEmitter {
    * @param {(mediaType: 'blobs' | 'icons') => Promise<string>} opts.getMediaBaseUrl
    * @param {import('./local-peers.js').LocalPeers} opts.localPeers
    * @param {Logger} [opts.logger]
+   * @param {boolean} [opts.isArchiveDevice] Whether this device is an archive device
```

Comment:
Is there ever a time we'd instantiate a project without this option? I think not, so maybe we can make this required?

Reply:
Addressed in 2dc69a6a0351cb1db0ff65b813e1b0efdccaccae